libs/image-editor/ImageEditor/src/main/kotlin/org/wordpress/android/imageeditor/preview/PreviewImageFragment.kt

```diff
@@ -43,7 +41,8 @@ import org.wordpress.android.imageeditor.utils.ToastUtils.Duration
 import org.wordpress.android.imageeditor.utils.UiHelpers
 import java.io.File
 
-class PreviewImageFragment : Fragment() {
+class PreviewImageFragment : Fragment(R.layout.preview_image_fragment) {
+    private var binding: PreviewImageFragmentBinding? = null
```

Comment:
@ParaskP7 ❓ Based on our conversation over on PR #14847, I thought we were going with:
```
    private var _binding: LayoutPickerPreviewFragmentBinding? = null    
    private val binding get() = _binding!!
```

Reply:
👋 @zwarm !

I think this was a misunderstand (a bit). Based on this [comment](https://github.com/wordpress-mobile/WordPress-Android/pull/14847#issuecomment-862248091) of mine, as a response on this PR #14847, I was saying that the since there is another such pattern in another `BottomSheet` related class, let's use that for `LayoutPreviewFragment` as well. However, seeing it more carefully now, this other place where this pattern is applied, the `SiteSettingsTimezoneBottomSheet`, it is actually wrong, as it ends up doing this `private val binding get() = _binding`, instead of including `!!`, which defeats the whole purpose of backing fields.

Since the whole project is utilizing the `private var binding: Xyz? = null` pattern without the extra backing field, I suggest we continue using that instead and use the `binding` field with an extra `?.` when needed. As such, I was comfortable introducing this new pattern for the `LayoutPreviewFragment` class and the PR #14847 you did, but **WRONGLY SO**, since the `SiteSettingsTimezoneBottomSheet` was actually using it wrongly. Apologies for the confusing, this is my bad.

I personally suggest we continue with what is already used everywhere and only update to the backing field pattern after we have discussed it and agreed that with everyone. Please check this `paqN3M-hV-p2#comment-510` comment from @renanferrari. If you want you can post another comment suggesting the new backing field pattern and see where it goes. Wdyt?

Reply:
@ParaskP7 - Still sounds like we are unsure of the direction we should use in the app; however I added a task to remove the backing field and just go with `binding?.`. I will also include a new task for adjusting  `SiteSettingsTimezoneBottomSheet` to get rid of the backing field. May as well adjust things as we find them. We could also update that post to note the changes we made. 

I will approve this PR and get it merged.

Reply:
👋 @zwarm !

Thanks for the update. 🙏 

> Still sounds like we are unsure of the direction we should use in the app;

Sure seems like it, but I believe we are doing a good work in trying to sync as much as possible, kudos to you for that! 🥇 

> I added a task to remove the backing field and just go with binding?.. I will also include a new task for adjusting SiteSettingsTimezoneBottomSheet to get rid of the backing field. May as well adjust things as we find them. We could also update that post to note the changes we made.

🙇 

> I will approve this PR and get it merged.

🚀